release.nix

```
@@ -14,6 +14,19 @@ let
   vuizvuiShortRev = vuizvuiSrc.shortRev or "abcdefg";
   vuizvuiVersion = "pre${toString vuizvuiRevCount}.${vuizvuiShortRev}";
 
+  # version of the nixos-unstable channel to get programs.sqlite from
+  programsSqliteVersion = "21.05pre269929.ff96a0fa563";
+  programsSqliteSha256 = "0ip26j2h11n1kgkz36rl4akv694yz65hr72q4kv4b3lxcbi65b3p";
+  programsSqlite = (import nixpkgs {}).fetchzip {
```

```suggestion
  programsSqlite = pkgsUpstream.fetchurl {
```

Note the `fetchurl` instead of `fetchzip` because the actual functionality of `fetchzip` is in `postFetch` anyway.

---

Yeah, we also don't need recursive hash.